### ISSUE TITLE
orcid should pull in or prompt for user info

### ISSUE THREAD

**Issue by satra** [dandi/dandiarchive#102]
The orcid addition should pull in user name and email address (if publicly available). if not available, the UI should prompt for that info.

`curl -X GET --header 'Accept: application/ld+json' 'https://pub.orcid.org/v2.1/0000-0002-5312-6729'`

**Comment by yarikoptic** [comment]
As a sailing on a shake down cruise, I can attest that it would be wonderful, and AFAIK should be quite simple (as @satra demonstrated) to at least populate name and affiliation .

**Comment by waxlamp** [comment]
This is something that would happen on the DLP, is that right? If an entity has an orcid, we can query for the username and email, but I don't understand the case of prompting for info--when does that happen?

**Comment by djarecka** [comment]
@waxlamp - I think this is related to our recent discussions. If orcid is available, we can perhaps use the names from the orcid

**Comment by waxlamp** [comment]
We need more clarity on how exactly this might work. I am still not sure what the original issue report is describing. When does that `curl` command happen? And what does the UI prompt for? @satra, could you weigh in on this?

**Comment by satra** [comment]
Indeed an integration with the orcid api would be better than our own
database.

Pull in is only because of vjsf being able to pull objects in.

**Comment by waxlamp** [comment]
So, when an ORCID is associated to a dandiset contributor, you would like to retrieve the ORCID record and autofill the contributor data with the results, is that right?

**Comment by satra** [comment]
yes, and vice verse the metadata editor could prompt for contributors already in dandi. essentially a more custom add contributor functionality eventually. also, we may desire contributor acknowledgement explicitly in the future. so this could be seen as a specific element of a contributor ui component.
